doc/rst/language/evolution.rst

```diff
@@ -18,7 +18,7 @@ version 1.28, September 2022
 ----------------------------
 
 Version 1.28 included some significant changes to the overload resolution
-rules. In addition, that version enables implicit conversion from
+rules. In addition, version 1.28 enables implicit conversion from
```

Comment:
I agree the previous wording was clunky, but might just use "it" for brevity / lack of repetition (seems sufficiently clear) and "enabled" in order to keep the tense similar.

Reply:
Fixed.